code/modules/power/supermatter/supermatter.dm

```diff
@@ -187,6 +187,12 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 
 	/// cooldown tracker for accent sounds,
 	var/last_accent_sound = 0
+	///Var that increases from 0 to 1 when a psycologist is nearby, and decreases in the same way
+	var/psyCoeff = 0
+	///A pinkish overlay used to denote the presance of a psycologist. We fade in and out of this depending on the amount of time they've spent near the crystal
+	var/obj/overlay/psy/psyOverlay = new /obj/overlay/psy/
```

Comment:
```suggestion
	var/obj/overlay/psy/psyOverlay = /obj/overlay/psy
```
Store it as a path and create it in `Initialize()`
Also there's no need for a `/` at the end of it

Reply:
Sure. I hope this was done correctly.